public/index.html

```diff
@@ -17,10 +17,14 @@
 
     <link rel="apple-touch-icon" href="./logo192.png" />
     <link rel="manifest" href="./manifest.json" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css">
+
```

Comment:
Do you need the CDNs? Bootstrap should be in node_modules.  Suggest to try the react-bootstrap carousel instead.  https://react-bootstrap.github.io/components/carousel/